src/Appwrite/Platform/Modules/Sites/Http/Deployments/Download/Get.php

```diff
@@ -133,8 +125,15 @@ public function action(
                 ->setStatusCode(Response::STATUS_CODE_PARTIALCONTENT);
 
             $response->send($device->read($path, $start, ($end - $start + 1)));
```

Comment:
Should we add here setContentType, and headerss x-peak, content-Disposition, and cache-control?

It would make it consustent with non-ranged requests